arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/continuations/ior.kt

```diff
@@ -0,0 +1,44 @@
+package arrow.core.continuations
+
+import arrow.continuations.generic.AtomicRef
+import arrow.continuations.generic.updateAndGet
+import arrow.core.EmptyValue
+import arrow.core.Ior
+import arrow.core.identity
+import arrow.typeclasses.Semigroup
+
+public suspend inline fun <E, A> ior(
```

Comment:
We should also expose `object ior` with `suspend operator fun invoke` and `fun eager` like the other builders

Reply:
Done 👍🏾